composer.json

```diff
@@ -98,7 +98,7 @@
         "friendsofphp/php-cs-fixer": "^3.8",
         "lusitanian/oauth": "^0.8",
         "magento/magento-coding-standard": "*",
-        "magento/magento2-functional-testing-framework": "^4.3.0",
```

Comment:
I think we should revert this change

Reply:
The changes in that package which are required for the correct functioning of the test suite here were released in version 4.3.2, so that's the minimum version we are compatible with. If we do not include this change, then the test-suite will consistently fail when using `composer update --prefer-lowest`.

https://github.com/magento/magento2-functional-testing-framework/pull/904#issuecomment-1598783540